x-pack/plugins/maps/server/data_indexing/indexing_routes.ts

```diff
@@ -129,7 +129,7 @@ export function initIndexingRoutes({
         const resp = await coreContext.elasticsearch.client.asCurrentUser.delete({
           index: request.body.index,
           id: request.params.featureId,
-          refresh: true,
+          refresh: true, // TODO: review stateless elasticsearch impact
```

Comment:
This is part of the geo editing feature. Users can delete an object from the map. That corresponds with the deletion of a document inside an ES-index.

The map will refresh after this deletion (running a `_search`. It needs to omit the feature that just has been deleted. That's why `refresh=true`. Could likely be `wait_for` as well.